docs/deploy.md

```diff
@@ -155,8 +155,9 @@ For development on the official homeworld servers (the LocalForward lines set up
                 LocalForward 5906 localhost:5906
                 LocalForward 5910 localhost:5910
 
-        # Note that you will need Kerberos tickets
-        # (generate them with kinit)
+        # Note that you will need Kerberos tickets.
+        # Generate them for your Kerberos identity from the root-admins section of setup.yaml, via
+        # kinit <Kerberos identity>
```

Comment:
Maybe it would be better to say `$ kinit <kerberos principal>`?

Reply:
Fixed.